pyvista/core/filters/image_data.py

```diff
@@ -1262,14 +1272,16 @@ def cells_to_points(self, scalars: str | None = None, *, copy: bool = True):
                 raise ValueError(
                     f"Scalars '{scalars}' must be associated with cell data. Got {field.name.lower()} data instead.",
                 )
-        return self._remesh_points_cells(points_to_cells=False, scalars=scalars, copy=copy)
+        return self._remesh_points_cells(
+            points_to_cells=False, scalars=scalars, resize_dims=resize_dims, copy=copy
+        )
 
     def _remesh_points_cells(
         self,
         points_to_cells: bool,
         scalars: str | None,
+        resize_dims: VectorLike[bool] | Literal['1D', '2D', '3D'],
         copy: bool,
-        remesh_singleton_dims: bool = False,
```

Comment:
This parameter will need to be deprecated (I can help with this after merging this PR with #6678)

Reply:
Whoops, looks like this was just a change in parameter name, so no deprecation is needed. I was thinking this was the `pad_singleton_dims` paramater for the `pad_image`. That one will need to be deprecated.